docker-compose.uffizzi.yml

```diff
@@ -38,14 +38,14 @@ services:
 
       DJANGO_ALLOWED_HOSTS: '*' # Change this in production
       DATABASE_URL: postgresql://postgres:password@localhost:5432/flagsmith
-      DISABLE_INFLUXDB_FEATURES: 'true' # set to false to enable InfluxDB
+      DISABLE_ANALYTICS_FEATURES: 'true' # set to false to enable InfluxDB
       PREVENT_SIGNUP: 'false'
       ENABLE_ADMIN_ACCESS_USER_PASS: 'true'
       DJANGO_SECRET_KEY: '**SoMeTeMpOrArYsEcReT!!'
 
       ENV:
-        dev # set to "prod" in production.
-
+        dev
+        # set to "prod" in production.
```

Comment:
Is there a reason this was pushed down onto the next line? Some kind of conflicting linting?

Reply:
Red Hat's YAML extension does this. Reverted.